redis-test/test_strings.py

```diff
@@ -141,3 +141,18 @@ def test_exists_multiple_existent_key():
     assert r.get(key3) == val3
     assert r.get(key4) == None
     assert r.exists(key1, key2, key3, key4) == 3
+
+def test_strlen():
+    r = connect()
+    key1 = random_string(10)
+    val1 = random_string(10)
+    key2 = random_string(10)
+    val2 = random_string(1000)
+    key3 = random_string(10)
+
+    assert r.set(key1, val1) == True
+    assert r.set(key2, val2) == True
+    r.delete(key3)
+    assert r.strlen(key1) == 10
+    assert r.strlen(key2) == 1000
+    assert r.strlen(key3) == 0
```

Comment:
Please add another test of what happens when you have a non-string value (e.g., a list) and you try to run strlen on it. I know what will happen in Scylla (since the list and string values come from a different array, strlen will return 0 exactly like the key is missing) but I want to know if the same happens also in Redis. (If not, we can make this test xfail - and it will remain xfail as long as we keep separate databases for the different types.)